core/src/kmx/kmx_plus.cpp

```diff
@@ -231,7 +245,7 @@ COMP_KMXPLUS_DISP::valid(KMX_DWORD _kmn_unused(length)) const {
     DebugLog("disp: baseCharacter str#0x%X", baseCharacter);
   }
   for (KMX_DWORD i=0; i<count; i++) {
-    DebugLog("disp#%d: to: str0x%X -> str0x%X", i, entries[i].to, entries[i].display);
+    DebugLoad("disp#%d: to: str0x%X -> str0x%X", i, entries[i].to, entries[i].display);
```

Comment:
I moved the most noisy cases, not all of them, and yes that was deliberate so that DebugLog still has some general value. Would you suggest a better solution?  I was trying to have it work like this:
- small amount of logging (DebugLog) at load time to give general info about the kbd
- verbose logging with DebugLoad() to dig into the actual contents (i.e. string table)
- very little logging during the critical path (keystrokes etc) for non-exceptional cases
- DebugTran() calls for verbose logging of transform path

Reply:
Sounds good to me